modules/oe_theme_helper/src/Plugin/Field/FieldFormatter/FeaturedMediaFormatter.php

```diff
@@ -0,0 +1,227 @@
+<?php
```

Comment:
It has been copied from #518 .

Reply:
Please don't copy over missing features or functionalities from other pull requests! Please remove all copied files and use EWPP-32 to rebase this branch over it.